content/en/docs/reference/using-api/cel.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Common Expression Language in Kubernetes
+reviewers:
+- jpbetz
+- cici37
+content_type: concept
+weight: 35
+min-kubernetes-server-version: 1.25
+---
+
+<!-- overview -->
+
+The [Common Expression Language (CEL)](https://github.com/google/cel-go) is used
+in the Kubernetes API to declare validation rules, policy rules, and other
+constraints or conditions.
+
+CEL expressions are evaluated directly in the
+{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}, making CEL a
+convenient alternative to out-of-process mechanisms, such as webhooks, for many
+extensibility use cases. Your CEL expressions continue to execute so long as the
+control plane's API server component remains available.
+
+<!-- body -->
+
+## Language Overview
+
+The [CEL
+language](https://github.com/google/cel-spec/blob/master/doc/langdef.md) has a
+straightforward syntax that is similar to the expressions in with C, C++, Java,
+JavaScript and Go.
+
+CEL was designed to be embedded into applications. Each CEL "program" is a
+single expression that evaluates to a single value. CEL expressions are
+typically short "one-liners" that inline well into the string fields of Kubernetes
+API resources.
+
+Inputs to a CEL program are "variables". Each Kubernetes API field that contains
+CEL declares in the API documentation which variables are available to use for
```

Comment:
`contains CEL` -> `contains CEL expression`? How about we refer `cel` as the language and `cel expression` as the expression we used to validate through the whole doc?

Reply:
This reads OK to me, kinda like how one might say a field contains JSON.  Technically it might be more precise to say "contains a JSON object declaration" each time, but it's clear what is being stated.